modules/redisearch.ts

```diff
@@ -97,12 +102,12 @@ export class Redisearch extends Module {
      * @returns Array reply, where the first element is the total number of results, and then pairs of document id, and a nested array of field/value.
      */
     async search(index: string, query: string, parameters?: FTSearchParameters): Promise<[number, ...Array<string | string[]>]> {
-        let args: string[] = [];//[index, query];
+        let args: string[] = [index, query];
```

Comment:
I am wondering why it was commented out and it still worked 🤔

Reply:
It was a copy of your search--unknown-argument--125 where you commented it out for testing purposes.